edge-util/src/Microsoft.Azure.Devices.Edge.Storage/InMemoryDbStore.cs

```diff
@@ -186,6 +262,8 @@ public ItemKeyedCollection(IEqualityComparer<byte[]> keyEqualityComparer)
                 .Select(i => (i.Key, i.Value))
                 .ToList();
 
+            internal IList<Item> AllItems => this.Items;
```

Comment:
Exposing the items through this property so that it can be used for serialization using ProtoBuf when backups are created. Added a comment to state the same.

`ItemList` creates another list that has a tuple type instead of using `Item` directly which makes it tricky to add ProtoBuf annotations to the type.

Reply:
Not needed anymore with the new changes on the PR